### PR TITLE
Fixes #162

### LINKED ISSUE #162: ValidationAssert with contains in-/valid satisfying specific condition
Having `containsValidSatisfying` and `containsInvalidSatisfying` methods on `ValidationAssert` would be useful

### PR DESCRIPTION
**Pull request by mchmielarz**

### CODE REVIEW

**Comment by mchmielarz** [comment]
@pivovarit Would you look at it too? :D

**Review by Grandys — COMMENTED** [review]
Few minor comments regarding documentation, code looks good 👍

**Comment by Opalo** [comment]
WUT :D What happens here? It's still not merged, after a year and a half? :D

**Comment by mchmielarz** [comment]
@Opalo I've been waiting for the review for whole this time (pun intended) ;)

File: src/main/java/org/assertj/vavr/api/AbstractTryAssert.java [hunk 1]
```diff
@@ -90,6 +90,9 @@ public SELF contains(VALUE expectedValue) {
      *
      * @param requirement to further assert on the object contained inside the {@link io.vavr.control.Try}.
      * @return this assertion object.
+     * @throws AssertionError       if the actual {@link Try} is null or empty.
```
[the remark this answers is not in the document]
Comment (mchmielarz) [reply src/main/java/org/assertj/vavr/api/AbstractTryAssert.java:93]: Sure, I'll update docs in this class to use simple names. In the following PRs I'll change this in other classes too.

**Review by Grandys — APPROVED** [review]
LGTM 👍

**Pull request merged**